### PR TITLE
Update to Rome testnet

### PR DESCRIPTION
**Pull request by kek-mex**

### CODE REVIEW

**Comment by kek-mex** [comment]
@blrhc I changed some words from using the UK spelling to the US spelling if you wonder :)

File: README.md [hunk 1]
```diff
@@ -171,6 +168,7 @@ Until the Joystream mainnet goes live, a sequence of test networks will be rolle
 | Athens          | 17.04.19          |   24.06.19    | [Link](/testnets/athens/README.md)        |
 | Sparta          | 28.02.19          |   29.03.19    |       N/A        |
 | Mesopotamia     | 21.12.18          |   28.02.19    |       N/A        |
+| Acropolis       | 24.06.19          |   13.03.20    | [Link](/testnets/acropolis/README.md)        |
```
Comment (bwhm) [inline README.md]: Please put this on top to keep it in order.

File: testnets/README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ It also contains templates for planning releases.
 | Athens          | 27.04.19          |   24.06.19    | [Link](athens)  |
 | Sparta          | 28.02.19          |   29.03.19    |       NA        |
 | Mesopotamia     | 21.12.18          |   28.02.19    |       NA        |
+| Acropolis       | 24.06.19          |   13.03.20    | [Link](acropolis)        |
```
Comment (bwhm) [inline testnets/README.md]: As comment above

File: testnets/rome/README.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ Table of Contents
 
 During the planning stages of a new testnet, we produce a new release plan.
 
-While working on [Acropolis](/testnets/acropolis), we decided to abandon the approach of creating a single `Release Project` kanban board on GitHub. These quickly became bloated, and created more confusion than value. We instead created [Tracking Issues](#tracking-issues) for the two major component of the release - the [forum](https://github.com/Joystream/joystream/issues/47) and the [storage node](https://github.com/Joystream/joystream/issues/57). These made it far simpler to get a good grasp of completed and outstanding items at a high level.
+While working on [Acropolis](/testnets/acropolis), we decided to abandon the approach of creating a single `Release Project` kanban board on GitHub. These quickly became bloated and created more confusion than value. We instead created [Tracking Issues](#tracking-issues) for the two major component of the release - the [forum](https://github.com/Joystream/joystream/issues/47) and the [storage node](https://github.com/Joystream/joystream/issues/57). These made it far simpler to get a good grasp of completed and outstanding items at a high level.
```
Comment (bwhm) [inline testnets/rome/README.md]: I think this comma is correct?

File: testnets/rome/README.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ Table of Contents
...
 
-After the [Acropolis Lessons Learned Meeting](/meetings/acropolis#lessons-learned), and thinking about our approach to project management and progress tracking, we have decided to expand on this concept. The release plan will no longer contain a "todo" list, but instead link to "live" [Tracking Issues](#tracking-issues) covering all aspects and actions of the release. Each of these will be as contained as they can, to avoid the need for all team members to be present and listening to in-depth technical and organizational conversations outside of their scope.
+After the [Acropolis Lessons Learned Meeting](/meetings/acropolis#lessons-learned), and thinking about our approach to project management and progress tracking, we have decided to expand on this concept. The release plan will no longer contain a "todo" list but instead link to "live" [Tracking Issues](#tracking-issues) covering all aspects and actions of the release. Each of these will be as contained as they can, to avoid the need for all team members to be present and listening to in-depth technical and organizational conversations outside of their scope.
```
Comment (bwhm) [inline testnets/rome/README.md]: I'm sure this comma is correct!

**Pull request merged**